lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,76 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    def request_params
```

Comment:
Does it actually work? I'd expect that this method needs to be in the `Audit::ListCommand`.

Reply:
It works as `Audit::ListComand` inherits `request_params` from `HammerCLIForeman::Comand`. So it is the same thing I guess, but it would be good to be specific that we are overriding the method for `Audit::ListCommand`.